### DISCUSSION TITLE
Releases, Packages and Environments blocs always Included in the home page

### DISCUSSION THREAD

**Discussion by cba85** [community/community#62378] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hello, I've found a bug recently.

Releases, Packages and Environments blocs are always included in the home page, even if I uncheck the options to remove them.

**Comment by StevenEWright** [comment, marked as answer]
Dupe of: https://github.com/orgs/community/discussions/62025
